whynotearth/foodouken.com
# Make phone number input only accept numbers and only shows numbers on mobile software keyboard

# Issue Being Addressed

None.

# Type of PR

Put an `x` in the brackets for the type(s) of PR this is. You may tick more than one.

[x] Update to Existing Feature

# Description

Desktop still functions the same way for phone number input, but now its easier to enter on mobile.


# How to Test/Reproduce

Steps to reproduce bad UX on current site:
1. Go to start an order on https://foodouken.com/ with a mobile device
2. Notice in phone number field you can input text
3. ![IMG_7914](https://user-images.githubusercontent.com/44442621/87003257-4b866280-c189-11ea-8f3c-c8154348b255.png)

THE FIX:
1. Go to start an order on https://deploy-preview-309--foodouken.netlify.app/ with a mobile device
2. Notice in phone number field you have big number buttons on the software keyboard
3. ![IMG_7917](https://user-images.githubusercontent.com/44442621/87003341-707ad580-c189-11ea-973e-46675e239415.png)

# Additional Context

This gives a better UX.
